### PR TITLE
Issue 66 - AWS ECR Credentials

### PR DESCRIPTION
**Pull request by chelomontilla**
This PR fix issue: [https://github.com/jetstack/version-checker/issues/66](https://github.com/jetstack/version-checker/issues/66)

The AWS sdk has by default many ways to configure the credentials, through environment variables, using kubernetes IRSA... 
I think the best option is to use the default credential provider chain so the final user can choose how to configure credentials. This solution also has the same option as before configuring AWS environment variables. 
Specifying Credentials: [https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)

```release-note
Environment variables VERSION_CHECKER_ECR_ACCESS_KEY_ID, VERSION_CHECKER_ECR_SECRET_ACCESS_KEY and VERSION_CHECKER_ECR_SESSION_TOKEN are no longer used. Now uses the AWS sdk default credential provider chain. More info: [https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html) [Specifying Credentials] section
```

### CODE REVIEW

**Comment by chelomontilla** [comment]
Hi,
this is taking to long to merge. Is there something I can do on my side to help?
Thanks.

**Comment by JoshVanL** [comment]
Thanks @chelomontilla , apologies for the delay on this.

/lgtm
/hold 
Let’s merge when we are ready to release v0.3.0

**Comment by alexandresavicki** [comment]
hey, any progress on this? When v0.3.0 will be released?

File: deploy/charts/version-checker/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -3,4 +3,7 @@ kind: ServiceAccount
 metadata:
   labels:
 {{ include "version-checker.labels" . | indent 4 }}
+{{- with .Values.serviceAccount.annotations }}
+{{ toYaml . | indent 8 }}
+{{- end }}
```
Comment (davidcollom) [inline deploy/charts/version-checker/templates/serviceaccount.yaml:8]:
```suggestion
{{- with .Values.serviceAccount.annotations }}
  annotations:
{{ toYaml . | indent 4 }}
{{- end }}
```

**Comment by davidcollom** [comment]
AWS SDK was updated in #182 - and was included in [v0.4.4](https://github.com/jetstack/version-checker/releases/tag/v0.4.4) and above

**Pull request closed without merging**